src/app/components/common/SearchBar.tsx

```
@@ -1,10 +1,14 @@
+import { MdSearch } from 'react-icons/md';
+
 export default function SearchBar({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)